servers/display_server.h

```diff
@@ -306,6 +306,9 @@ class DisplayServer : public Object {
 		}
 		return scale;
 	}
+
+	virtual void ios_set_ui_state(bool hide_home_indicator, bool suppress_bottom_ui_gesture, bool suppress_top_ui_gesture, bool hide_status_bar = true, float status_bar_fade_time = 0.2);
```

Comment:
```suggestion
	virtual void ios_set_ui_state(bool p_hide_home_indicator, bool p_suppress_bottom_ui_gesture, bool p_suppress_top_ui_gesture, bool p_hide_status_bar = true, float p_status_bar_fade_time = 0.2);
```
Prefix arguments with `p_`